### PR TITLE
Fix/hardcoded fees

### PR DESCRIPTION
**Pull request by xenide**

### CODE REVIEW

File: src/abis/IVexchangeV2Pair.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
+export default {
```
Comment (OliverNChalk) [inline src/abis/IVexchangeV2Pair.ts]: Generally prefer named exports, objections to:
```suggestion
export PAIR_ABI = {
```

**Review by OliverNChalk — APPROVED** [review]
Looks good, feel pretty comfy because of the constants - means we should have caught all usages assuming Uniswap did a good job of using the constants

File: src/fetcher.ts [hunk 2]
```diff
@@ -61,22 +62,29 @@ export abstract class Fetcher {
...
-    const { reserve0, reserve1 } = reserves;
+
+    const reserves = (await getReservesMethod.call()).decoded
+    const { reserve0, reserve1 } = reserves
     const balances = tokenA.sortsBefore(tokenB) ? [reserve0, reserve1] : [reserve1, reserve0]
-    return new Pair(new TokenAmount(tokenA, balances[0]), new TokenAmount(tokenB, balances[1]))
+
+    const swapFeeABI = find(IVexchangeV2Pair.abi, { name: 'swapFee' })
+    const method = connex.thor.account(pairAddress).method(swapFeeABI)
+    const swapFee = JSBI.BigInt((await method.call()).decoded['0'])
```
Comment (KennethAshley) [inline src/fetcher.ts]: I'd prefer to have the await on a different line here. Its doing multiple things in this one line

**Review by KennethAshley — CHANGES_REQUESTED** [review]
One change everything else looks good!

File: src/fetcher.ts (hunk 2 above)
Comment (xenide) [reply src/fetcher.ts]: fixed, please see if it's okay now

File: src/abis/IVexchangeV2Pair.ts (hunk 1 above)
Comment (xenide) [reply src/abis/IVexchangeV2Pair.ts]: fixed

**Pull request merged**